src/day8/re_frame/trace/app_state.cljs

```diff
@@ -0,0 +1,51 @@
+(ns day8.re-frame.trace.app-state
+  (:require [reagent.core :as r]
+            [clojure.string :as str]
+            [devtools.formatters.core :as cljs-devtools]
+
+            [cljs.pprint :refer [pprint]]))
+
+(defn string->css [css-string]
+  (->> (map #(str/split % #":") (str/split (get css-string "style") #";"))
```

Comment:
This could be made slightly clearer by squeezing a bit more out of the threading macro:

```clojure
(defn string->css [css-string]
  (->> (str/split (get css-string "style") #";")
       (map #(str/split % #":"))
       (reduce (fn [acc [property value]]
                 (assoc acc (keyword property) value)) {})))
```